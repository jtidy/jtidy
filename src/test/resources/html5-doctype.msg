<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test html5-doctype-->
<messages>
  <message>
    <code>110</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Doctype given is ""]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like HTML5]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[no warnings or errors were found]]></text>
  </message>
</messages>
