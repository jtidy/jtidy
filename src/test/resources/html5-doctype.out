<!DOCTYPE html>
<html>
<head>
<title>HTML 5 Doctype Test</title>
</head>
<body>
<h1>This is a page with HTML 5 doctype.</h1>
</body>
</html>